convert/jenkinsjson/json/fileOperations_test.go

```diff
@@ -0,0 +1,181 @@
+package json
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	harness "github.com/drone/spec/dist/go"
+	"github.com/google/go-cmp/cmp"
+)
+
+type testRunner struct {
+	name  string
+	input Node
+	want  *harness.Step
+}
+
+func TestConvertFileOpsCreate(t *testing.T) {
+	// Get the working directory
```

Comment:
what does this test do?

Reply:
this is just reading a json and compare the string right? Let's remove this.

Reply:
@jamie-harness Do we need to remove and push PR again? or since it is already merged I will take care this in next PRs, leave this as is?